Week_5/Week_5/Challenge_5-1.py

```diff
@@ -0,0 +1,14 @@
+def upper_or_lower_count(string):
+    count_upper = 0
+    count_lower = 0
+    for letter in string:
+        if letter == " ":

```

Comment:
Can you explain this part to me?

Reply:
It's to account for the whitespace in the string. I realised that omitting that line of code made the program count the white space as part of uppercase.